episodes/01-raster-structure.Rmd

```diff
@@ -53,11 +52,10 @@ data values as stored in a raster and how R handles these elements.
 
 We will continue to work with the `dplyr` and `ggplot2` packages that were introduced
 in the [Introduction to R for Geospatial Data](https://datacarpentry.org/r-intro-geospatial/) lesson. We will use two additional packages in this episode to work with raster data - the
-`raster` and `rgdal` packages. Make sure that you have these packages loaded.
+`raster` and `sf` packages. Make sure that you have these packages loaded.
```

Comment:
should this text specifying the `raster` package be replaced with `terra`?

Reply:
You're right. I changed the text from `raster` to `terra`. Thanks!

Reply:
Thank you for your contribution on transitioning to terra. A few notes below on updating text content for learners to follow along with updated code will be very helpful:

Line 84 references GDAL() function that has now been removed and should be updated to reflect describe()
Line 151 references raster() and should be updated to reflect the forcing of dataframe creation
Lines 319-332 references raster() and nlayers()

Reply:
Thank you for reporting the typos. I fixed them.